Whiskey/en-US/about_Whiskey_SetVariableFromPowerShellDataFile.help.txt

```diff
@@ -0,0 +1,90 @@
+TOPIC
+    about_Whiskey_about_Whiskey_SetVariableFromPowerShellDataFile_Task
+
+DESCRIPTION

```

Comment:
By design. About topics are actually just text files. Unlike comment-based help, there's no parse-able structure to about topics.

Reply:
I mentioned this because it looks like it does **some** basic parsing with the Synopsis? E.g
```
C:\Build\> help about_Obj*
Name                              Category  Module                    Synopsis
----                              --------  ------                    --------
about_Objects                     HelpFile                            Provides essential information about objects in Windows PowerShell.
about_Object_Creation             HelpFile                            Explains how to create objects in Windows PowerShell.
```

Reply:
It looks like the parser grabs the text after the second header. It doesn't care what the header's text is.